biom/table.py

```diff
@@ -3343,6 +3353,14 @@ def to_hdf5(self, h5grp, generated_by, compress=True):
         compress : bool, optional
             Defaults to ``True`` means fields will be compressed with gzip,
             ``False`` means no compression
+        format_fs : dict, optional
+            Specify custom formatting functions for metadata fields. This dict
+            is expected to be {'metadata_field': function}, where the function
+            signature is (h5py.Group, str, dict, bool) corresponding to the
+            specific group the metadata dataset will be associated with, the
+            category being operated on, the metadata for the entire axis being
+            operated on, and whether to enable compression on the dataset.
+            Anything returned by this function is ignored.
```

Comment:
Might add a note that although the function does not return anything, it typically will create one or more new data sets (as is done in the tests). I was slightly confused how this was supposed to work until I looked at the test code.